docs/04-github-repositories.md

```diff
@@ -12,6 +12,8 @@ The following details a list of relevant Sygma repositories.
 | [**sygma-solidity**](https://github.com/sygmaprotocol/sygma-solidity) | Smart contracts used by Sygma bridge | Public repository, [audited](audits/); beta. |
 | [**sygma-relayer**](https://github.com/sygmaprotocol/sygma-relayer) | Go-implemented relayer service | Public repository, [audited](audits/); beta. |
 | [**sygma-sdk**](https://github.com/sygmaprotocol/sygma-sdk) | Tools to easily integrate any JS application with Sygma bridge | Public repository; production.|
-| [**sygma-ui**](https://github.com/sygmaprotocol/sygma-ui) | <p>Two part application that uses sygma-sdk to interact with blockchains:</p><ol><li>TransferUI is used to interact with Sygma smart contracts</li><li>ExplorerUI is used to track and navigate Sygma ecosystem</li></ol> | Public repository; production. |
+| [**sygma-ui**](https://github.com/sygmaprotocol/sygma-ui) | Application TransferUI that uses sygma-sdk to interact with Sygma smart contracts and blockchains | Public repository; production. |
```

Comment:
@haochizzle can we set this also to be in beta state

Reply:
done